### DISCUSSION TITLE
lightweight is important for sumatrapdf

### DISCUSSION THREAD

**Discussion by AHK0** [sumatrapdfreader/sumatrapdf#3333] [General]
seems many people like its lightweight.
in this case, why do not set the language packs as separate files? in that case, everyone keeps only the language he/she likes, probably this may further lose some weight of the software itself.

**Comment by kjk** [comment]
Separating per language would be too much work.

One option would be to not include translations in the .exe at all and download them only if the user changes the language.

Then again, strings are about 660 kB which is nothing in today's world. I have to balance small size with complexity of the code.
